packages/test-utils/src/utils.ts

```diff
@@ -91,7 +107,7 @@ export const setupTestEnv = (web3: any) => {
   return { provider, unlockedAccount };
 };
 
-export function signMessage(message, wallet): [number, string, string] {
+function signMessage(message, wallet): [number, string, string] {
```

Comment:
Why privatize these? Are they not being used anywhere else?

Reply:
Yup, they're nont